release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -65,6 +65,7 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 * With this update, you can export your application in the ZIP file format to another project or cluster using the *Export application* option on the *+Add* page. 
 * With this update, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 * With this update, you can set the default resource preference in the *User Preferences* -> *Applications* page. In addition, you can select another resource type to be the default.
+* With this update, you can make the `status.HostIP` node IP for pods visible in the *Details* tab of the *Pods* page.
```

Comment:
@opayne1, out of scope, but I wonder if it makes sense to parallelize these and drop the redundant _With this update,_ now that there are four of these in a row?

We've been using _IP address_ instead of just _IP_ when possible.

Reply:
Good point! I have a plan to reorg the web console sections and break up with admin/dev rel notes to make clearer. I will address that in that PR and i can share with dev perspective writers. 

I will update with IP address. I was looking for a word to make that sentence flow better. Thanks!